content/blog/high-utilization-in-systems/index.md

```diff
@@ -0,0 +1,92 @@
+
+Notice how all of those areas benefit the organization for the long-term when done right. It's not a coincidence that they are deprioritized.
+
+## Delivery
+
+If doing things this way impacts delivery speed and quality in the long run, why do we still see high utilization as a positive signal? My theory is that most of our industry is caught up playing short term games. 
+
+Leadership is expected to provide results in short-term cycles, hence management gets recognized for delivering projects in tight timelines and developers are rewarded when they are able to ship tasks as fast as possible. None of these behaviours are responsible for creating better products, teams or culture in the long run. This reminds me of a quote from Dave Packard:
+
+> “More Organisations Die Of Indigestion Than Starvation”
+
+This quote really resonates with me. We need teams focused on a small number of objectives that are aligned with the company vision. High utilization doesn't allow for strategic planning, getting teams caught up in a tactical mindset.
+
+## Things to look out for
+
+If you are not sure whether your team is under high utilization or not (you probably know) you can look for the following characteristics:
+
+* Team members working alone or in small teams due too many initiatives
+* Poor engagement in meetings
+* Asynchronous channels (Slack, email, etc) drying up in participation or being slow/ignored
+* Pull request lead cycle taking a long time
+    - PRs are taking longer to be opened (high complexity/lower quality in the codebase)
+    - PRs are taking longer to be reviewed (context switching) 
+* Increased amount of bugs in production
+* Knowledge silos
+* Lack of documentation
+
+Pay special attention to decisions on how the strategic objectives should be delivered being made without or far from the team that is executing it.
+
+## Caveats
+
+There are moments where high utilization is unavoidable. For example, if your company is trying to find market fit or when time to market is critical for success. These tend to be highly overestimated by companies, but it is a legitimate scenario that should be taken into consideration. With that said, keeping teams under high utilization for long periods of time is not a viable strategy.
+
+## Conclusion
+
+It takes a team to push against the myth of high utilization. Developers should raise their concerns when they feel they are lacking support or stretched too thin. Knowing when to saying "no" is a powerful skill to have. Managers should listen to the team's feedback, but also keep an eye on and push back when they feel like the team is juggling between too many responsibilities. You will rarely hear a "no" from junior developers and it is your job to pay attention to a high utilization situation. Leadership on the other hand should trust that their teams are making good decisions on how their strategic objectives should be delivered.
```

Comment:
> Leadership on the other hand should trust that their teams are making good decisions on how their strategic objectives should be delivered.

What if they don't? One could raise the concern that operating on "trust" in a business context might be an unrealistic expectation.

Reply:
Usually they don't and you have one of the two scenarios:

* A manager that is really stressed out trying to prevent this from leaking to the team
* A manager that let's the pressure goes down and abides to anything leadership says, now you have a very stressed out team

Reply:
> One could raise the concern that operating on "trust" in a business context might be an unrealistic expectation.

It's a great way to fail or increase attrition to not trust the people that are closest to the domain to execute correctly. If someone was hired you can assume that there's at least an implicit amount of trust on that person to execute until proven wrong.

Reply:
I should've been more clear. "They" refers to the teams, rather than leadership, meaning what if the teams are not making good decisions. However, I think your response answers that question as well: Lack of trust from leadership is not going to solve that issue either.

Reply:
If the team is not making good decisions that pressure will definitely come from the top. Either the vision/priorities are not well defined or managers are not doing a great job working with product to prioritize projects that contribute to the vision. But that's a tricky subject. 😄

Reply:
Prioritizing the many priorities is usually recognized by a [wanna be] high-achiever leadership. Something like "I'm too busy with 20 different projects and tasks, but I know which ones to prioritize to keep my lead happy" is usually well received and appreciated. Sometimes it's temporary and reasonable, but it has long-term side effects.

Some days I give myself a pat on the back that I finished multiple tasks in parallel; but I try to remind myself that I'm not necessarily multitasking; instead, I might be failing multiple task simultaneously! with the cost of:  

-   missing the teamwork opportunities.
-   losing some context.
-   focusing on my individual priorities (rather than team priorities) and unconsciously ignoring other's priorities.
-   Quality of outcome.
-   Ignoring/deprioritizing less flashy and recognized tasks/activities like knowledge sharing and documentation, team building and enablement, and focusing on the greater goals and collective impact.